home/templates/home/base.html

```diff
@@ -1,82 +1,79 @@
 		<button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-		  <span class="navbar-toggler-icon"></span>
+			<span class="navbar-toggler-icon"></span>
 		</button>
 		<div class="collapse navbar-collapse" id="navbarSupportedContent">
-		  <ul class="navbar-nav me-auto mb-2 mb-lg-0">
-			<li class="nav-item">
-			  <a class="nav-link active" href="#">Explore</a>
-			</li>
-			<li class="nav-item">
-			  <a class="nav-link" href="#">Tags</a>
-			</li>
-			 <li class="nav-item">
-			  <a class="nav-link" href="#">Users</a>
-			</li>
-			</li>
-			 <li class="nav-item">
-			  <a class="nav-link" href="{% url 'about' %}"</a>About</a>
-			</li>
-		  </ul>
-		  <nav class="col-5 navbar-light mt-3">
-		  <form class="d-flex">
-			<input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
-			<button class="btn btn-light" type="submit">Search</button>
-		  </form>
-		  </nav>
+			<ul class="navbar-nav me-auto mb-2 mb-lg-0">
+				<li class="nav-item">
+					<a class="nav-link active" href="#">Explore</a>
+				</li>
+				<li class="nav-item">
+					<a class="nav-link" href="#">Tags</a>
+				</li>
+				<li class="nav-item">
+					<a class="nav-link" href="#">Users</a>
+				</li>
+				<li class="nav-item">
+					<a class="nav-link" href="{% url 'about' %}"</a>About</a>
+				</li>
+			</ul>
+			 <nav class="col-5 navbar-light me-auto mt-3">
+			  <form class="d-flex">
+				<input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+				<button class="btn btn-light" type="submit">Search</button>
+			  </form>
+			  </nav>
 			<!-- Navbar Right Side -->
-			<div class="navbar-nav ">
+			<div class="navbar-nav">
 			  {% if user.is_authenticated %}
-			  <ul class="nav navbar-nav navbar-right">
-			    <li><p class="nav-item m-2"><b>Hello {{user.username}}</b></p></li>
+			  <ul class="nav navbar-nav" >
+				<li><p class="nav-item m-2"><b>Hello {{user.username}}</b></p></li>
 				<li><a class="nav-item nav-link m-2 " href="#"><b>Profile</b></a></li>
 				<li><a class="nav-item nav-link m-2" href="#"><b>Logout</b></a></li>
 			  </ul>
 			  {% else %}
 			  <p class="nav-item nav-link m-2"><b>Hello visitor</b></p>
-				<a class="nav-item nav-link m-2" href="#">
-					<strong>
-					Login
-					</strong>
-				</a>
-				<a class="nav-item nav-link m-2" href="#">
-					<strong>
-					Register
-					<strong>
-				</a>
+				<a class="nav-item nav-link m-2" href="#"><b>Login</b></a>
+				<a class="nav-item nav-link m-2" href="#"><b>Register<b></a>
 			  {% endif %}
 			</div>
 		</div>
-	</div>
-</nav>
-    <main role="main" class="container">
-      <div class="row">
-        <div class="col-md-8">
-          {% block content %}{% endblock %}
-        </div>
-      </div>
-    </main>
```

Comment:
I seem to remember there was an important reason why one of the bootstrap scripts needed to be loaded after the rest of the page was already loaded, so not sure why are you removing this.

Reply:
Ok, I moved them back